packages/request-logic/src/request-logic.ts

```diff
@@ -601,4 +657,86 @@ export default class RequestLogic implements RequestLogicTypes.IRequestLogic {
       this.advancedLogic,
     );
   }
+
+  /**
+   * Computes the diff between the confirmed and pending request
+   *
+   * @param requestStateConfirmed the confirmed request state
+   * @param requestStatePending the pending request state
+   * @returns an object with the pending state attributes that are different from the confirmed one
+   */
+  private computeDiffBetweenPendingAndConfirm(
```

Comment:
```suggestion
  private computeDiffBetweenPendingAndConfirmed(
```
If name is not changed